packages/edit-post/src/components/sidebar/post-status/index.js

```diff
@@ -64,6 +79,7 @@ export default function PostStatus() {
 					<>
 						<PostStatusPanel />
 						<PostFeaturedImagePanel withPanelBody={ false } />
+						{ showPostExcerptPanel && <PrivatePostExcerptPanel /> }
```

Comment:
I believe this check should probably move within the component itself like the other PostSomething components.

Reply:
>// Post excerpt panel is rendered in different place depending on the post type.
				// So we cannot make this check inside the PostExcerpt component based on the current edited entity.

Do you think there is a better way by adding some kind of flag or something?

Reply:
Why is the post excerpt rendered in different places depending on the post type?

Reply:
Based on the designs: https://github.com/WordPress/gutenberg/issues/59689#issuecomment-1991723289. I think the main reason is that in the entities that we treat excerpt as a `description` feels better above in the card where we essentially describe the entity. In regular post types excerpt is more of part of the entity with a different functionality - describe the content.

Reply:
Are we really helping ourselves by creating these inconsistencies?